approval_polls/static/approval_polls/create.js

```diff
@@ -18,4 +18,35 @@ $(function () {
 
   $('[data-toggle=tooltip]').tooltip();
   $('button#add-choice').click($.proxy(this.addChoiceField, this));
+
+  function roundMinutes(date) {
+    if (date.getMinutes() > 0 && date.getMinutes() < 30){
+      date.setHours(date.getHours());
+      date.setMinutes(30);
+    }
+    else if (date.getMinutes() > 30 && date.getMinutes() < 60){
+      date.setHours(date.getHours()+1);
+      date.setMinutes(0);
+    }
+    return date;
+  }
+
+  var minDateTimelogic = function( currentDateTime ){
+    this.setOptions({
+      minDate: roundMinutes(currentDateTime),
+      minTime: currentDateTime,
```

Comment:
There seems to be an issue here. 'currentDateTime' is the time set by the datetimepicker each time. Thus each time a datetime is selected, the previous dates are disabled automatically. This is also allowing me to select a datetime from the previous month, while not allowing me to select dates from the next month. 
I believe you should be passing in `new Date()` instead so that only the current time is used to compute the mintime.